src/plugins/Sticky.ts

```diff
@@ -8,12 +10,14 @@ class StickyPlugin extends GenericPlugin {
     const { container } = this.ad;
 
     const stickybit = stickybits(container);
+    dispatchEvent(this.ad.id, LOG_LEVELS.INFO, 'Sticky', `Sticky container added to ad.`);
```

Comment:
Lets do "Sticky Plugin" as the creator. Same for all plugins and networks as X Network